datacenter/ucp/3.0/guides/admin/install/install-on-azure.md

```diff
@@ -0,0 +1,286 @@
+      docker4x/az-nic-ips
+    ```
+
+## Set up IP configurations on an Azure virtual machine scale set
+
+Configure IP Pools for each member of the VM scale set during provisioning by
+associating multiple `ipConfigurations` with the scale set’s
+`networkInterfaceConfigurations`. Here's an example `networkProfile`
+configuration for an ARM template that configures pools of 32 IP addresses
+for each VM in the VM scale set.
+
+```json
+"networkProfile": {
+  "networkInterfaceConfigurations": [
+    {
+      "name": "[variables('nicName')]",
+      "properties": {
+        "ipConfigurations": [
+          {
+            "name": "[variables('ipConfigName1')]",
+            "properties": {
+              "primary": "true",
+              "subnet": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
+              },
+              "loadBalancerBackendAddressPools": [
+                {
+                  "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/loadBalancers/', variables('loadBalancerName'), '/backendAddressPools/', variables('bePoolName'))]"
+                }
+              ],
+              "loadBalancerInboundNatPools": [
+                {
+                  "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/loadBalancers/', variables('loadBalancerName'), '/inboundNatPools/', variables('natPoolName'))]"
+                }
+              ]
+            }
+          },
+          {
+            "name": "[variables('ipConfigName2')]",
+            "properties": {
+              "subnet": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
+              }
+            }
+          }
+          .
+          .
+          .
+          {
+            "name": "[variables('ipConfigName32')]",
+            "properties": {
+              "subnet": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
+              }
+            }
+          }
+        ],
+        "primary": "true"
+      }
+    }
+  ]
+}
+```
+
+## Install UCP 
+
+Use the following command to install UCP on the manager node.
+The `--pod-cidr` option maps to the IP address range that you configured for
+the subnets in the previous sections, and the `--host-address` maps to the
+IP address of the master node.
```

Comment:
Do we need to even note the `--host-address`? Unless the VM has >1 NIC it'll default to the one, correct? If left in we should note "private IP address of the master node" or "internal IP address of the master node" as folks will try to use a Public IP here

Reply:
@stevenfollis I do believe the bootstrapper defaults to an interface, but I find that it's safer to specify since it's not so uncommon for a VM to have multiple interfaces. Adding the "private address of the master node" makes sense.

Reply:
Makes sense, thanks @mark-church 👍

Reply:
If this conversation is resolved, please mark it as such. Thanks!